src/super_gradients/common/auto_logging/auto_logger.py

```diff
@@ -1,108 +1,79 @@
-import json
-import os
 import logging
-import pkg_resources
+import os
+import sys
+from typing import Optional
 
 
 class AutoLoggerConfig:
     """
-    A Class for the Automated Logging Config that is created from the JSON config file (auto_logging_conf)
+    A Class for the Automated Logging Config
     """
 
-    @staticmethod
-    def generate_config_for_module_name(
-        module_name,
-        training_log_path=None,
-        log_level=logging.INFO,
-        max_bytes=10485760,
-        logs_dir_path=None,
-        handlers_list=None,
-    ) -> dict:
-        """
-        generate_config_for_module_name - Returns a Config Dict For Logging
-            :param module_name:     The Python Module name to create auto_logging for
-            :param log_level:       Minimal log level to set for the new auto_logging
-            :param max_bytes:       Max size for the log file before rotation starts
-            :param handlers_list:    A list specifying the handlers (Console, etc..) - Better Leave Empty or None
-            :param training_log_path: Path to training log file which all modules of super_gradients will write to. Ignored
-             when set to None.
-            :param logs_dir_path: Path to sg_logs directory (default=None), where module logs will be saved. When set
-                to None- module logs will be saved in ~/sg_logs (created if path does not exist). Main use case is for
-                testing.
-
-
-            :return: python dict() with the new auto_logging for the module
-        """
-
-        # LOADING THE ORIGINAL ROOT CONFIG FILE
-        conf_file_name = "auto_logging_conf.json"
-        conf_file_path = os.path.join(
-            pkg_resources.resource_filename("super_gradients", "/common/auto_logging/"), conf_file_name
+    FILE_LOGGING_LEVEL = os.environ.get("LOG_LEVEL", "DEBUG").upper()
+    CONSOLE_LOGGING_LEVEL = os.environ.get("LOG_LEVEL", "INFO").upper()
+
+    filename: str = None
+
+    @classmethod
+    def setup_default_logging(self, log_level: str = None) -> None:
+        self.setup_logging(
+            filename=os.path.expanduser(f"~/sg_logs/last_{os.getppid()}.log"),
+            copy_already_logged_messages=False,
+            log_level=log_level,
+        )
+
+    @classmethod
+    def setup_logging(self, filename: str, copy_already_logged_messages: bool, log_level: str = None) -> None:
+        if copy_already_logged_messages and self.filename is not None and os.path.exists(self.filename):
+            with open(self.filename, "r", encoding="utf-8") as src:
+                with open(filename, "w") as dst:
+                    dst.write(src.read())
```

Comment:
I think that we should first check if the directory exists with something like:
Path(filename).mkdir(parents=True, exist_ok=True)

OR do it only in "setup_default_logging", because when we call this function from "base_sg_logger" we can assume that "self._local_dir" already exists

Reply:
Agree, thanks for the suggestion.